integrations/bitdefender/actions/get-company-details.ts

```diff
@@ -0,0 +1,38 @@
+import type { NangoAction, ProxyConfiguration } from '@nangohq/shared';
+import type { BitdefenderCompanyResponse } from '../types.js';
+import { toCompany } from '../mappers/to-company.js';
+
+export default async function runAction(nango: NangoAction) {
+    // Documentation URL: https://www.bitdefender.com/business/support/en/77209-126239-getcompanydetails.html
```

Comment:
documentation should go above the endpoint property. Remove the prefix "Documentation URL"